CHANGELOG.md

```diff
@@ -12,6 +12,9 @@ x.x.x
 
 **CHANGES**
 - Remove support for Python 3.6.
+- Changes to FSx created by ParallelCluster:
```

Comment:
`FSx for Lustre file systems created [...]`

Reply:
Done